roboflow/util/prediction.py

```diff
@@ -473,7 +497,9 @@ def json(self):
         return prediction_group_json
 
     @staticmethod
-    def create_prediction_group(json_response, image_path, prediction_type, image_dims):
+    def create_prediction_group(
```

Comment:
why this method is `static`? The correct way for factory methods is `classmethods`

Reply:
I think this prediction.py could use a large refactor in general, I will not attempt to defend it in this PR